### ISSUE TITLE
oldv does not work on macos

### ISSUE THREAD

**Issue by Le0Developer** [vlang/v#22991]
### Describe the bug

oldv is not passing `-no-parallel` when compiling the `cmd/v` using `vc` which panics

### Reproduction Steps

1. Be on macOS
2. Run `VERBOSE=1 v run cmd/tools/oldv master`

### Expected Behavior

A V executable in ~/.cache/oldv/v_at_master/v

### Current Behavior

The build panics (invisibly without `VERBOSE=1`).

### Possible Solution

Add `VFLAGS=-no-parallel` when invoking oldv.

### Additional Information/Context

Discord discussion: https://discord.com/channels/592103645835821068/592294828432424960/1311322309159551017

### V version

V 0.4.8 c1df71a.844d89f

### Environment details (OS name and version, etc.)

V full version: V 0.4.8 c1df71a.844d89f
OS: macos, macOS, 15.1.1, 24B2091
Processor: 10 cpus, 64bit, little endian, Apple M4

getwd: /Users/leodev/v
vexe: /Users/leodev/v/v
vexe mtime: 2024-11-27 12:40:43

vroot: OK, value: /Users/leodev/v
VMODULES: OK, value: /Users/leodev/.vmodules
VTMP: OK, value: /tmp/v_501

Git version: git version 2.47.0
Git vroot status: weekly.2024.48-8-g844d89fd (1 commit(s) behind V master)
.git/config present: true

CC version: Apple clang version 16.0.0 (clang-1600.0.26.4)
emcc version: N/A
thirdparty/tcc status: thirdparty-macos-arm64 713692d4

<sub><a href="https://huly.app/guest/vlang-66f40c4d-a476b54c67-771fdd?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NzQ3MjQ5NmQyZWY0Y2JjYzQ2YmNiYWIiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InctYWxleGFuZGVyLXZsYW5nLTY2ZjQwYzRkLWE0NzZiNTRjNjctNzcxZmRjIn0._9mF8a4QXzKAxPj1NDRUR-pn5Mw3rzfJBFALsChNOZ0">Huly&reg;: <b>V_0.6-21432</b></a></sub>

**Comment by spytheman** [comment]
```
VJOBS=1 v run cmd/tools/oldv.v master
```
... is another workaround.

The env variable only needs to be set the first time it is run, after that the `v` executable will be reused, so the bootstrapping issue will be sidestepped.

**Comment by spytheman** [comment]
i.e. after the first run, you can later do: `v run cmd/tools/oldv.v master -c './v version'`, and for example the following works:
```
0[16:07:28]@m1: (master) /opt/v $ VJOBS=1 cmd/tools/oldv HEAD^^^
#################  context.commit_v: HEAD^^^ #####################
#     v commit hash: 8bc94dd41f | folder: /Users/delyanangelov/.cache/oldv/v_at_HEAD___
0[16:07:59]@m1: (master) /opt/v $ cmd/tools/oldv HEAD^^^ -c './v version'
#################  context.commit_v: HEAD^^^ #####################
#     v commit hash: aeaf607259 | folder: /Users/delyanangelov/.cache/oldv/v_at_HEAD___
#           command: ./v version
#            result: V 0.4.8 8bc94dd
```

**Comment by spytheman** [comment]
Fixed in latest V 0.4.8 2c3ad8d .
